### PR TITLE
hpux support for basic port properties

### PR DESCRIPTION
**Pull request by Anirudh-Gupta**
Gives listening ports, its address and protocol

### CODE REVIEW

File: lib/resources/port.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # author: Dominik Richter
 
 require 'utils/parser'
-
+require 'pry'
```
Comment (chris-rock) [inline lib/resources/port.rb]: Do we need `pry`?

File: test/unit/mock/cmd/netstat-inet6 [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Active Internet connections (IPv6, including servers)
```
Comment (chris-rock) [inline test/unit/mock/cmd/netstat-inet6]: It may make sense to rename the file to `test/unit/mock/cmd/hpux-netstat-inet6`

**Comment by chris-rock** [comment]
@Anirudh-Gupta Thanks for the addition. I have only small questions. Do we require `pry`? Also, I would like to prefix the test-files with hpux in this case, since `netstat` is available on multiple systems. Do you think that makes any sense?

**Comment by Anirudh-Gupta** [comment]
@chris-rock Yes it does. Thanks for pointing it out.

**Comment by chris-rock** [comment]
Awesome @Anirudh-Gupta

**Pull request merged**